homeassistant/components/touchlinesl/config_flow.py

```diff
@@ -0,0 +1,64 @@
+"""Config flow for Roth Touchline SL integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from pytouchlinesl import Module, TouchlineSL
+from pytouchlinesl.client import RothAPIError
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
+from .const import CONFIG_ENTRY_VERSION, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_USERNAME): str,
+        vol.Required(CONF_PASSWORD): str,
+    }
+)
+
+
+class TouchlineSLConfigFlow(ConfigFlow, domain=DOMAIN):
```

Comment:
Do we have something unique we can fetch from the API we can use to avoid users setting up the same account twice (which we won't want since then unique_ids collide and hell breaks loose)

Reply:
Yep, I think we can use the username in this case.